fuzzers/libfuzzer_nocmp/fuzzer.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for libFuzzer fuzzer."""
+
+import os
+
+from fuzzers import utils
+from fuzzers.libfuzzer import fuzzer as libfuzzer_fuzzer
+
+
+def build():
```

Comment:
nit: similar to run_fuzzer, you can have a run_build with extra_cflags argument. i guess we will do more libfuzzer experiments, so good to keep libfuzzer's fuzzer.py that way.

Reply:
I think we plan on using runtime variants for this.